src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -32,10 +36,59 @@
     @MockBean
     private OcrDataValidator ocrDataValidator;
 
+    @MockBean
+    private AuthService authService;
+
+    @Test
+    void should_throw_unauthenticated_exception_when_auth_header_is_missing() throws Exception {
+        String requestBody = readResource("ocr-data/valid/valid-ocr-data.json");
+        given(authService.authenticate("")).willThrow(UnauthenticatedException.class);
```

Comment:
this is mocked, so instead of `should_throw_unauthenticated_exception_when_auth_header_is_missing` perhaps `should _return_401_status_when_auth_service_throws_unauthenticated_exception`?